From: Dermot Bradley <dermot_bradley@yahoo.com>
Date: Fri, 22 Jan 2021 22:07 +0000
Subject: [PATCH] step-certificates: Fix 32bit builds

Bump go-piv to v1.7.0 for x32 overflow fix.

Upstream PR #441. This patch has already been merged in upstream master
and so can be removed when next version is released.

---

diff -aur a/go.mod b/go.mod
--- a/go.mod
+++ b/go.mod
@@ -7,7 +7,7 @@
 	github.com/Masterminds/sprig/v3 v3.1.0
 	github.com/aws/aws-sdk-go v1.30.29
 	github.com/go-chi/chi v4.0.2+incompatible
-	github.com/go-piv/piv-go v1.6.0
+	github.com/go-piv/piv-go v1.7.0
 	github.com/golang/mock v1.4.4
 	github.com/google/uuid v1.1.2
 	github.com/googleapis/gax-go/v2 v2.0.5
